apps/codeforafrica/src/lib/index.js

```diff
@@ -556,7 +556,8 @@ function getAboutPageStaticProps() {
         },
         {
           slug: "our-partners",
-          partners: getOurPartners("about"),
+          // No action button on About Page
+          partners: { ...getOurPartners("about"), action: null },
```

Comment:
```suggestion
          partners: { ...getOurPartners("about") },
```

Should be initialized to null in `getOurPartners`.